templates/Gemfile.erb

```diff
@@ -39,7 +39,7 @@ group :development, :test do
   gem "dotenv-rails"
   gem "factory_girl_rails"
   gem "pry-rails"
-  gem "rspec-rails", "~> 3.1.0"
+  gem "rspec-rails", "~> 3.3.0"
```

Comment:
I'd be in favor of changing it to `>= 3.3.0` myself, but I think that's a change for a different PR.